### DISCUSSION TITLE
Can we update to Reanimated 3?

### DISCUSSION THREAD

**Discussion by eduardo-santos-tribia** [react-navigation/react-navigation#11256] [Q&A]
Hi, I saw that [Reanimated 3 was released](https://blog.swmansion.com/releasing-reanimated-3-0-17fab4cb2394) 2 days ago and I want to know if it's "safe" to update to this version using the latest versions of `react-navigation` and react native `v0.71.3`? 

Also, there's [this issue](https://github.com/react-navigation/react-navigation/issues/11252) opened regarding react-navigation + reanimated 3...

**Comment by kacperkapusciak** [comment]
Hi @eduardo-santos-tribia 👋 

It's as safe as Reanimated 3 is stable.

With each major version of a package it's pretty common to have some bugs which authors didn't catch on the internal testing. There's already [a 3.0.1 patch version](https://github.com/software-mansion/react-native-reanimated/releases/tag/3.0.1) which fixes some of them.

The`@react-navigation/drawer` didn't need any changes on the `react-navigation` side as Reanimated 3 uses the same Reanimated 2 API.

As there's no migration needed on the developer side (so `yarn add react-native-reanimated@latest` costs nothing) you can always upgrade and revert to previous version if necessary 😅 

Cheers

**Reply by eduardo-santos-tribia** [reply]
Thanks!

**Comment by swrobel** [comment]
We've had crashes because of [this issue](https://github.com/software-mansion/react-native-reanimated/issues/4177). We don't use reanimated directly - only via react-navigation/stack, so it's preventing us from upgrading.

**Comment by SteveGreenley** [comment]
I've had problems using Reanimated 3 with React Navigation. A simple app using the drawer navigator displays a white screen when you open the drawer. Downgrading to Reanimated 2 fixed the issue.
